ocs_ci/deployment/acm.py

```diff
@@ -190,19 +190,23 @@ def download_downstream_binary(self):
             f'--path="/dist/subctl-{version_str}*-linux-{binary_pltfrm}.tar.xz":/tmp --confirm'
         )
         run_cmd(cmd)
-        # Wait till image extract happens and subctl dir appears
-        time.sleep(30)
         decompress = (
             f"tar -C /tmp/ -xf /tmp/subctl-{version_str}*-linux-{binary_pltfrm}.tar.xz"
         )
-        run_cmd(decompress)
+        p = subprocess.run(decompress, stdout=subprocess.PIPE, shell=True)
+        if p.returncode:
+            logger.error("Failed to untar subctl")
+            raise CommandFailed
+        else:
+            logger.info(p.stdout)
         target_dir = os.path.expanduser("./bin")
```

Comment:
Let's use bin_dir from config where we have it defined under RUN section:
```
RUN:
  bin_dir: /home/jenkins/bin
```
But for local runs it can be different path as we overwrite in in custom config file in jenkins to ~/bin

Reply:
done